hyperopt/spark.py

```diff
@@ -468,6 +469,7 @@ def run_task_thread():
             params = self._get_spec_from_trial(trial)
 
             def run_task_on_executor(_):
+                import traceback
```

Comment:
please add imports on the top of the file only. Ideally we should use isort and black as well for nicely formatted files